src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onSelectedTabId }) => {
+  const currentTab = tabs.find(tab => selectedTabId === tab.id);
```

Comment:
current tab will be undefined in case wrong selectedTabId
![2023-08-03_15h02_02](https://github.com/mate-academy/react_tabs-js/assets/61904995/2f66f39d-0b76-4e0e-a949-36c1ed6f081f)